src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -215,36 +215,33 @@
             type: command
             short-summary: Create diagnostic settings for the specified resource.
             parameters:
+                - name: --name
+                  short-summary: The name of the diagnostic settings.
                 - name: --resource-id
                   type: string
                   short-summary: The identifier of the resource.
                 - name: --resource-group -g
                   type: string
-                  short-summary: Name of the resource group.
+                  short-summary: Name of the resource group for the Log Analytics, and Storage Account when the name of
```

Comment:
This is confusing to me. Is this supposed to say "Log Analytics and Storage Account"? The way it read it makes it sound like `--resource-group` is used for the storage account name if a full resource ID isn't given.

Reply:
ok, will update.